File: Step1/Logical-Thinking/Pattern16/Solution.cpp
#include <iostream>
using namespace std;
void pattern(int n){
        int letter = 65;
    for(int i = 1; i<=n; ++i){
        for(int j = 0; j<i; ++j){
           cout<<(char(letter));
        }
        ++letter;
        cout<<endl;
    }
}
int main()
{
    pattern(5);

    return 0;
}

File: Step1/Logical-Thinking/Pattern16/Solution.java
// SAMPLE pattern
// A 
// BB 
// CCC 
// DDDD
// EEEEE
public class Main
{
    static void pattern(int n){
        int letter = 65;
    for(int i = 1; i<=n; ++i){
        for(int j = 0; j<i; ++j){
            System.out.print((char)letter);
        }
        ++letter;
        System.out.println();
    }
}
	public static void main(String[] args) {
		pattern(5);
	}
}
